app/views/principals/pre_qualification_form.html.erb

```diff
@@ -1,47 +1,111 @@
 <div class="l-constrained">
   <section class="l-pre-qualification-questions">
-    <%= heading_tag(t('registration.heading'), level: 1, class: 'l-pre-qualification-questions__heading') %>
-    <p class="l-pre-qualification-questions__subheading"><%= t('registration.subheading') %></p>
-
-    <%= form_for @prequalification, url: prequalify_principals_path, html: { class: 'l-pre-qualification-questions__form' } do |f| %>
-      <fieldset class="l-pre-qualification-questions__question-group t-question-1">
-        <legend class="l-pre-qualification-questions__question"><%= t('registration.question_one.heading') %></legend>
-        <p><%= t('registration.question_one.explanation') %></p>
-        <label class="l-pre-qualification-questions__label">
-          <%= f.radio_button :question_1, 1, class: 'l-pre-qualification-questions__input', checked: true %>
-          <%= t('registration.answer_yes') %>
-        </label>
-        <label class="l-pre-qualification-questions__label">
-          <%= f.radio_button :question_1, 0, class: 'l-pre-qualification-questions__input' %>
-          <%= t('registration.answer_no') %>
-        </label>
-      </fieldset>
-
-      <fieldset class="l-pre-qualification-questions__question-group t-question-2">
-        <legend class="l-pre-qualification-questions__question"><%= t('registration.question_two.heading') %></legend>
-        <p><%= t('registration.question_two.explanation') %></p>
-        <label class="l-pre-qualification-questions__label">
-          <%= f.radio_button :question_2, 1, class: 'l-pre-qualification-questions__input', checked: true %>
-          <%= t('registration.answer_yes') %>
-        </label>
-        <label class="l-pre-qualification-questions__label">
-          <%= f.radio_button :question_2, 0, class: 'l-pre-qualification-questions__input' %>
-          <%= t('registration.answer_no') %>
-        </label>
-      </fieldset>
-
-      <fieldset class="l-pre-qualification-questions__question-group t-question-3">
-        <legend class="l-pre-qualification-questions__question"><%= t('registration.question_three.heading') %></legend>
-        <p><%= t('registration.question_three.explanation') %></p>
-        <label class="l-pre-qualification-questions__label">
-          <%= f.radio_button :question_3, 1, class: 'l-pre-qualification-questions__input', checked: true %>
-          <%= t('registration.answer_yes') %>
-        </label>
-        <label class="l-pre-qualification-questions__label">
-          <%= f.radio_button :question_3, 0, class: 'l-pre-qualification-questions__input' %>
-          <%= t('registration.answer_no') %>
-        </label>
-      </fieldset>
+    <%= heading_tag(t('registration.heading'), level: 1) %>
+    <p class="l-pre-qualification-questions__intro-text"><%= t('registration.introduction') %></p>
+
+    <%= form_for @prequalification, url: prequalify_principals_path, html: { class: 'form' } do |f| %>
+      <%= f.form_row :question_1 do %>
```

Comment:
Appreciate they're radio buttons but belt and braces - I'd imagine if none were selected (somehow, or if defaults changed later) then the server would return an error